src/components/Buildbaron/BBCreatedTickets.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { useQuery } from "@apollo/client";
+import {
+  GetCreatedTicketsQuery,
+  GetCreatedTicketsQueryVariables,
+} from "gql/generated/types";
+import { GET_CREATED_TICKETS } from "gql/queries/get-created-tickets";
+import { BBTitle, TitleAndButtons } from "./BBComponents";
+import { BuildBaronTable } from "./BuildBaronTable";
+
+interface Props {
+  taskId: string;
+  dispatchBanner;
+  setCreatedTicketsCount;
+  createdTicketsCount;
+}
+
+export const CreatedTickets: React.FC<Props> = ({
+  taskId,
+  dispatchBanner,
+  setCreatedTicketsCount,
+  createdTicketsCount,
+}) => {
+  const { data, startPolling, stopPolling } = useQuery<
+    GetCreatedTicketsQuery,
+    GetCreatedTicketsQueryVariables
+  >(GET_CREATED_TICKETS, {
+    variables: { taskId },
+    onError(error) {
+      dispatchBanner.errorBanner(
+        `There was an error getting tickets created for this task: ${error.message}`
+      );
+    },
+  });
+  const length = data?.bbGetCreatedTickets?.length;
+
+  if (createdTicketsCount > data?.bbGetCreatedTickets?.length) {
+    startPolling(1 * 1000);
```

Comment:
You can just put `1000` instead of `1*1000` they both produce the same result

Reply:
They do. I can do that. I personally like using 1*1000 so that it's clear that it's one second (I'm assuming seconds). Similar to how the pollInterval constant does 60*1000 even though it can just be 60000. Lmk if you want me to change that, I don't have a very strong opinion, just a preference.

Reply:
That makes sense, What do you think about extracting this to be some universal constant like some variable SECOND. And we can use that to replace all our instances of 1000 in poll intervals so we can do `1 * SECOND` or `60 * SECOND`